src/python/detectors/detect_activated_debug_feature/detect_activated_debug_feature.py

```diff
@@ -0,0 +1,17 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=detect-activated-debug-feature@v1.0 defects=1}
+def detect_activated_debug_feature_noncompliant():
+    from django.conf import settings
+    # Noncompliant: The debug feature is enabled.
+    settings.configure(DEBUG=True)
+# {/fact}
+
+
+# {fact rule=detect-activated-debug-feature@v1.0 defects=0}
+def detect_activated_debug_feature_compliant():
+    from django.conf import settings
```

Comment:
The coding style should remain consistent, i.e. here it is imported in a function, and in untrusted-deserialization, the imports are in the file

Reply:
Here in untrusted-deserialization.py ,the import to flask and app cannot be given inside the function as app is used just before the function starts at line number 10 .

Reply:
Here in untrusted-deserialization.py ,the import to flask and app cannot be given inside the function as app is used just before the function starts at line number 10 .